packages/grid/x-data-grid-pro/src/hooks/features/columnResize/useGridColumnResize.tsx

```diff
@@ -234,66 +261,6 @@ export const useGridColumnResize = (
     apiRef.current.publishEvent('columnResize', params, nativeEvent);
   });
 
-  const handleColumnResizeMouseDown: GridEventListener<'columnSeparatorMouseDown'> =
-    useEventCallback(({ colDef }, event) => {
-      // Only handle left clicks
-      if (event.button !== 0) {
-        return;
-      }
-
-      // Skip if the column isn't resizable
-      if (!event.currentTarget.classList.contains(gridClasses['columnSeparator--resizable'])) {
-        return;
-      }
-
-      // Avoid text selection
-      event.preventDefault();
-
-      logger.debug(`Start Resize on col ${colDef.field}`);
-      apiRef.current.publishEvent('columnResizeStart', { field: colDef.field }, event);
-
-      colDefRef.current = colDef as GridStateColDef;
-      colElementRef.current =
-        apiRef.current.columnHeadersContainerElementRef?.current!.querySelector<HTMLDivElement>(
-          `[data-field="${colDef.field}"]`,
-        )!;
-
-      const headerFilterRowElement = apiRef.current.headerFiltersElementRef?.current;
-
-      if (headerFilterRowElement) {
-        headerFilterElementRef.current = headerFilterRowElement.querySelector<HTMLDivElement>(
-          `[data-field="${colDef.field}"]`,
-        ) as HTMLDivElement;
-      }
-
-      colGroupingElementRef.current = findGroupHeaderElementsFromField(
-        apiRef.current.columnHeadersContainerElementRef?.current!,
-        colDef.field,
-      );
-
-      colCellElementsRef.current = findGridCellElementsFromCol(
-        colElementRef.current,
-        apiRef.current,
-      );
-
-      const doc = ownerDocument(apiRef.current.rootElementRef!.current);
-      doc.body.style.cursor = 'col-resize';
-
-      resizeDirection.current = getResizeDirection(event.currentTarget, theme.direction);
-
-      initialOffsetToSeparator.current = computeOffsetToSeparator(
-        event.clientX,
-        colElementRef.current!.getBoundingClientRect(),
-        resizeDirection.current,
-      );
-
-      doc.addEventListener('mousemove', handleResizeMouseMove);
-      doc.addEventListener('mouseup', handleResizeMouseUp);
-
-      // Fixes https://github.com/mui/mui-x/issues/4777
-      colElementRef.current.style.pointerEvents = 'none';
```

Comment:
This was preventing the double-click from going through, I removed it in favor of the other method we tried in that PR.